dali/base/dasds.cpp

```diff
@@ -5308,26 +5310,31 @@ class CStoreHelper : implements IStoreHelper, public CInterface
         }
     }
 
-    void writeStoreInfo(const char *base, const char *location, unsigned edition, unsigned *crc, CStoreInfo *storeInfo=NULL)
+    void writeStoreInfo(const char *base, const char *location, unsigned edition, unsigned *crcXml, unsigned *crcBinary, CStoreInfo *storeInfo = nullptr)
     {
         StringBuffer path, filename;
         filename.append(base).append('.').append(edition);
-        if (location) path.append(location);
+        if (location)
+            path.append(location);
         path.append(filename);
 
         OwnedIFile iFile = createIFile(path.str());
         OwnedIFileIO iFileIO = iFile->open(IFOcreate);
-        if (crc)
-            iFileIO->write(0, sizeof(unsigned), crc);
+        if (crcXml)
+        {
+            iFileIO->write(0, sizeof(unsigned), crcXml);
+            if (crcBinary)
+                iFileIO->write(sizeof(unsigned), sizeof(unsigned), crcBinary);
+        }
         if (storeInfo)
             storeInfo->cache.set(filename.str());
         iFileIO->close();
     }
 
-    void updateStoreInfo(const char *base, const char *location, unsigned edition, unsigned *crc, CStoreInfo *storeInfo=NULL)
+    void updateStoreInfo(const char *base, const char *location, unsigned edition, unsigned *crcXml, unsigned *crcBinary, CStoreInfo *storeInfo=NULL)
     {
         clearStoreInfo(base, location, edition, storeInfo);
-        writeStoreInfo(base, location, edition, crc, storeInfo);
+        writeStoreInfo(base, location, edition, crcXml, crcBinary, storeInfo);
```

Comment:
not critical to do now perhaps, but for consistency, it would be good to update refreshInfo and CStoreInfo, to look and load the binaryCrc too.